### PR TITLE
[a11y] IAP logo copy svg to text

### LINKED ISSUE #4719: ⌨ Replace copy on IAP logo image with `h1` HTML element
## Description
Permanent solution to issue #4718.

## TODO
- [x] confirm how this is going to react on devices with the IAP team, and whether we need to import the unique font used in the logo first (@BrookeCameron and @MCommonda)

## Screenshot of Issue
Screen Shot 2022-11-10 at 5 14 46 PM

## Acceptance Criteria
- [ ] Remove the [image of text](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-text-presentation.html) for the IAP logo & associated text.  
- [ ] Make "IT Apprenticeship Program for Indigenous Peoples" regular text and an `<h1>` (since there isn't one currently on the page). 
- [ ] Make "Apply today to get started on your IT career journey" regular text.

### PR DESCRIPTION
**Pull request by mnigh**
🤖 Resolves #4719.

h/t @esizer 

## 👋 Introduction

This PR moves the copy out of the IAP logo and into plain text to make it better accessible by assistive technology. It also makes it easier to add additional languages without having to create a new SVG for each locale.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Navigate to `/en/indigenous-it-apprentice/`
2. Observe logo on different viewports
3. Switch to French 
4. Observe logo on different viewports
5. Switch to Mi'kmaq
6. Observe logo on different viewports

## 📸 Screenshot

Screen Shot 2023-05-12 at 12 07 15

### CODE REVIEW

**Comment by substrae** [comment]
@mnigh @BrookeCameron 

Looking into this further, I think we will have to go with what I suggested in terms of a fixed width icon that center aligns with the text as it wraps. Having an image retain its aspect ratio and resize based on available height determined by variable text within a fixed container isn't possible, as far as I can tell.

File: apps/web/src/pages/Home/IAPHomePage/Home.tsx [hunk 1]
```diff
@@ -97,29 +82,57 @@ const Home = () => {
...
           data-h2-padding="base(x1.2, x2, x1, x2) p-tablet(0)"
           data-h2-layer="base(1, relative) p-tablet(1, absolute)"
           data-h2-order="base(1) p-tablet(2)"
           data-h2-location="p-tablet(5%, auto, auto, 50%)"
-          data-h2-width="base(100%) p-tablet(40vw)"
+          data-h2-width="base(100%) p-tablet(40em) l-tablet(55em)"
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column) p-tablet(row)"
+          data-h2-align-items="base(center) p-tablet(flex-start)"
+          data-h2-justify-content="base(center)"
```
Comment (yonikid15) [inline apps/web/src/pages/Home/IAPHomePage/Home.tsx:95]: I think adding ` data-h2-align-items="base(center)"` would make this look even better, but that's just me.

**Review by yonikid15 — DISMISSED** [review]
Looks fantastic! Nice work 👍

**Comment by substrae** [comment]
Alright, so I've made a handful of changes in my last commit.

- I've adjusted the way the whole section displays across smaller devices to provide more vertical space for the logo
- I've adjusted the layout on mobile and desktop to account for a consistently sized icon across languages
- @yonikid15 `align-items: center` was definitely added 👍 
- @BrookeCameron please feel free to review the screens below, though they generally reflect what we've already looked at

<details>
  <summary>Phones</summary>
  
</details> 

<details>
  <summary>Portrait tablets</summary>
  
</details> 

<details>
  <summary>Landscape tablets</summary>
  
</details> 

<details>
  <summary>Larger displays</summary>
  
</details>

**Comment by BrookeCameron** [comment]
This looks great! thanks for everyones hard work on this @mnigh @substrae @yonikid15 🎉✨🤘🏼

**Review by esizer — APPROVED** [review]
LGTM 👍

**Review by yonikid15 — APPROVED** [review]
⭐

**Pull request merged**